docs/requirements.txt

```diff
@@ -1,3 +1,8 @@
 jaxlib
 ipykernel
 nbsphinx
+# The next packages are for readthedocs
+matplotlib
+# Must install jax itself for notebook execution to work
+.
```

Comment:
It looks like we could also add an `install` section to the yml file:
https://docs.readthedocs.io/en/stable/config-file/v2.html#packages

Reply:
I don't have a strong preference here, this looks fine to me.